immunopepper/immunopepper.py

```diff
@@ -70,7 +70,7 @@ def parse_arguments(argv):
 
     general = parser_diff.add_argument_group('MISCELLANEOUS')
     general.add_argument("--compressed",help="compress the output files",action="store_true",default=False)
-    general.add_argument("--verbose", type=int, help="specify the output verbosity. Level 1 records the output file path.", required=False, default=1)
+    general.add_argument("--verbose", type=int, help="specify the output verbosity (0 - warn, 1 - info, 2 - debug) [1]", required=False, default=1)
```

Comment:
consider refactoring the repeated "general" option block into a function

Reply:
ok, done